From eaa0ead19ca43fcfc35bff12074cefd6317e2ee9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Hugo=20Beauz=C3=A9e-Luyssen?= <hugo@beauzee.fr>
Date: Wed, 9 Nov 2016 10:29:17 +0100
Subject: [PATCH 9/9] wincrypt: Unconditionally include dpapi.h

Since it now contains some functions that can be used on win10
---
 mingw-w64-headers/include/wincrypt.h | 2 --
 1 file changed, 2 deletions(-)

diff --git a/mingw-w64-headers/include/wincrypt.h b/mingw-w64-headers/include/wincrypt.h
index 759758f..7425823 100644
--- a/mingw-w64-headers/include/wincrypt.h
+++ b/mingw-w64-headers/include/wincrypt.h
@@ -5439,9 +5439,7 @@ WINIMPM HCERTSTORE WINAPI CertOpenStore (LPCSTR lpszStoreProvider, DWORD dwEncod
 }
 #endif
 
-#if WINAPI_FAMILY_PARTITION (WINAPI_PARTITION_DESKTOP)
 #ifndef _WINCRYPT_NO_DPAPI
 #include <dpapi.h>
 #endif
 #endif
-#endif
-- 
2.9.3

